### PR TITLE
First version of the Introduction Directory

### PR DESCRIPTION
**Pull request by lancehalvorsen**
- This has a copy of the Overview Guide in it. I'm curious how it feels to have it here, and can delete it if it seems strange.

- The Installation Guide is yet to come, but I wanted to get this much out there to see if there is any feedback.

### CODE REVIEW

File: introduction/A_overview.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+The aim of this introductory guide is to present a brief, high level overview of Phoenix, the parts that make it up, and the layers underneath that support it.
+
+### Phoenix
+
+Phoenix is actually the top layer of a multi-layer system designed to be modular and flexible. The other layers are the Elixir middleware project, Plug, and the Erlang web server, Cowboy. Plug and Cowboy are covered in the next sections of this guide.
+
+Phoenix is made up of a number of distinct parts, each with its own purpose and role to play in building a web application. We will cover them all in depth throughout these guides, but here's a quick breakdown.
+
+- The Endpoint
```
Comment (pel-daniel) [inline introduction/A_overview.md]: hey Lars, this part has no format. Is just rendering as a big list

File: introduction/A_overview.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+[Plug](http://hexdocs.pm/plug/) is Elixir's web middleware layer. Conceptually, it shares a lot with other middleware layers like Rack for Ruby or WSGI for Python. Plugs are reusable modules that share the same very small, very regular public api. They provide discrete behaviors - like request header parsing or logging. Because the Plug api is so consistent, they can be stacked and executed in a set order, like a pipeline. They can also be re-used within a project or across projects.
+
+Plugs can be written to handle almost anything, from authentication to parameter pre-processing, and even rendering.
+
+Phoenix takes great advantage of Plug in general - the router and controllers especially so.
+
+One of the most important things about Plug, is that it provides adapters to HTTP servers which will ultimately deliver application content to your users. Currently, Plug only provides an adapter to Cowboy, which we will talk about next, but there are plans to provide adapters for other servers in the future.
+
+Links to more in-depth information on Plug can be found in the [Resources Guide](http://www.phoenixframework.org/docs/resources).
```
Comment (pel-daniel) [inline introduction/A_overview.md]: What do you think about linking to the specific section of the page ? (i.e. http://www.phoenixframework.org/docs/resources#section-plug). The same for the link of cowboy below.

Comment (lancehalvorsen) [inline introduction/A_overview.md]: Thanks. I should probably link it to the [Plug Guide](http://www.phoenixframework.org/docs/understanding-plug) itself.

File: introduction/A_overview.md (hunk 1 above)
Comment (lancehalvorsen) [inline introduction/A_overview.md]: That's true, but it does have the same treatment as the other topics on the page which have sup-topics - Cowboy and System Dependencies. Consistency as a virtue? :)

Comment (lancehalvorsen) [inline introduction/A_overview.md]: Ah, I see what you mean now. The formatting got lost in the copy/paste. Please disregard my previous comment.

Comment (pel-daniel) [inline introduction/A_overview.md]: Cool you see it, sorry for my poor explanation

**Comment by lancehalvorsen** [comment]
- added new installation guide
- changed Overview and Up and Running to correspond to those changes

This is "feature complete" at this point.

**Pull request merged**